tests/test_level/test_level_11.py

```diff
@@ -1,5 +1,6 @@
 import hedy
 import textwrap
+from hedy import Command
```

Comment:
This is the only change in this file. Is this import really necessary?

Reply:
Thanks for pointing it out! It is unnecessary and I removed it.